ocs_ci/ocs/acm/acm.py

```diff
@@ -152,6 +155,46 @@ def install_submariner_ui(self, globalnet=True):
             for s in get_non_acm_cluster_config()
             if s.MULTICLUSTER["multicluster_index"] != primary_index
         ][0]
+        # submariner catalogsource creation
+        if config.ENV_DATA["submariner_release_type"] == "unreleased":
+            submariner_downstream_unreleased = templating.load_yaml(
+                constants.SUBMARINER_DOWNSTREAM_UNRELEASED
+            )
+            # Update catalog source
+            submariner_full_url = "".join(
+                [
+                    constants.SUBMARINER_DOWNSTREAM_UNRELEASED_BUILD_URL,
+                    config.ENV_DATA["submariner_version"],
+                ]
+            )
+            curl_cmd = f"curl --retry 3 --retry-delay 5 -Ls {submariner_full_url}"
+            jq_cmd1 = (
+                'jq -r \'[.raw_messages[].msg | select(.pipeline.status=="complete") |'
+                "{{nvr: .artifact.nvr, index_image: .pipeline.index_image}}] | .[0]'"
+            )
+
+            jq_cmd2 = "jq -r '.index_image.\"v4.14\"'"
```

Comment:
we should not hardcode the version but rather take from ocp version from config ,
Maybe we can directly update jq command to get the value from `.index_image.v4.14`

Reply:
completely replaced with python code, thanks to @prsurve

Reply:
done